Shared/Mirror/mirrorReflection.js

```diff
@@ -0,0 +1,36 @@
+//This script acts on the reflection box in front of the mirror so that when and avatar
+//enters the area, the mirror will reflect an image via the spectator camera
+
+(function () {
+    print("Starting reflection script...............");
+    var mirrorID, reflectionAreaID;
+   //get id of reflection area and mirror
+    this.preload = function(entityID) {
+        reflectionAreaID = entityID;
+        mirrorID = Entities.getEntityProperties(reflectionAreaID, 'parentID').parentID;
+        print("Reflection area ID is : " + reflectionAreaID);
+        print("Mirror area ID is : " + mirrorID);
+  };
+
+    //when avatar enters reflection area, begin reflecting
+    this.enterEntity = function(entityID){
+        
+      // var mirrorOverlayOn = function() {
```

Comment:
remove unused code for production, you will be able to find it back in your git history in case you need it again